packages/web/src/features/manageClubTable/frames/ManageClubTableMainFrame.tsx

```diff
@@ -0,0 +1,402 @@
+          width: "100%",
+          marginBottom: "8px",
+          textAlign: "right",
+          color: "#555555",
+        }}
+        type="p"
+      >{`총 ${mockData.length}개`}</Typography>
+      <TableFrameInner>
+        <TableRowFrameInner
+          style={{
+            gridTemplateColumns: headers
+              .map(headerInfo => headerInfo.headerWidth)
+              .join(" "),
+          }}
+        >
+          {headers.map((headerInfo, index) => (
+            <TableCellInner
+              style={
+                headerInfo.headerType === "HeaderSort"
+                  ? {
+                      gridColumn: `${index + 1} / ${index + 2}`,
+                      cursor: "pointer",
+                    }
+                  : { gridColumn: `${index + 1} / ${index + 2}` }
+              }
+              key={headerInfo.headerName}
+              onClick={
+                headerInfo.headerType === "HeaderSort"
+                  ? () => setSortColumnName(headerInfo.headerName)
+                  : () => {}
+              }
+            >
+              <TableCell type={headerInfo.headerType} width="100%">
+                {headerInfo.headerName}
+              </TableCell>
+            </TableCellInner>
+          ))}
+        </TableRowFrameInner>
+
+        {data.map((rowInfo, rowIndex) => (
+          <TableRowFrameInner
+            style={{
+              gridTemplateColumns: headers
+                .map(headerInfo => headerInfo.headerWidth)
+                .join(" "),
+              borderBottom: "1px #EEEEEE solid",
+            }}
+            id={`row${rowIndex.toString()}`}
+          >
+            {Object.entries(rowInfo).map(([key, value], index) => (
+              <TableCellInner
+                style={{
+                  gridColumn: `${index + 1} / ${index + 2}`,
+                }}
+                key={key}
+              >
+                <TableCell type="Default" width="100%">
+                  {key === "status" ? (
+                    <Tag
+                      color={
+                        ManageClubTagColors[
+                          Object.keys(ManageClubRentalBusinessStatus)[
+                            Object.values(
+                              ManageClubRentalBusinessStatus,
+                            ).indexOf(value)
+                          ] as keyof ManageClubTagColorsInterface
+                        ]
+                      }
+                    >
+                      {value}
+                    </Tag>
+                  ) : (
+                    formattedString(key, value)
+                  )}
+                </TableCell>
+              </TableCellInner>
+            ))}
+          </TableRowFrameInner>
+        ))}
+      </TableFrameInner>
+      {mockData.length > 1 ? (
+        <NoticePagination
+          totalPage={Math.ceil(mockData.length / 10)}
+          currentPage={page}
+          limit={10}
+          setPage={setPage}
+        />
+      ) : null}
```

Comment:
삼항연산자 사용 시 뒤에꺼가 `null`이면 `&&` 사용하는 걸 권장(?)합니다!! (그냥 제 개인 취향임)

```suggestion
      {mockData.length > 1 && (
        <NoticePagination
          totalPage={Math.ceil(mockData.length / 10)}
          currentPage={page}
          limit={10}
          setPage={setPage}
        />
      )}
```